From 9cb58a6fc554b4c005bef7df29cb254dca18935c Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Thu, 10 Sep 2015 12:47:37 +0100
Subject: [PATCH] Remove trailing ':' from THD engine failure message

Running thermald on a machine with no thermal zones or sensors one
sees the final log messages as follows:

ML zone: invalid sensor type TSKN
Zone update failed: unable to bind
No Thermal Zones found
No thermal sensors found
THD engine start failed:

..the final message ends with a ':' making one wonder if there
are any following messages, which there are none.  Remove the trailing
':' as it is confusing to the user.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 src/thd_engine_default.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/thd_engine_default.cpp b/src/thd_engine_default.cpp
index d10ae59..1ebee6e 100644
--- a/src/thd_engine_default.cpp
+++ b/src/thd_engine_default.cpp
@@ -629,7 +629,7 @@ int thd_engine_create_default_engine(bool ignore_cpuid_check,
 	// Initialize thermald objects
 	thd_engine->set_poll_interval(thd_poll_interval);
 	if (thd_engine->thd_engine_start(ignore_cpuid_check) != THD_SUCCESS) {
-		thd_log_error("THD engine start failed:\n");
+		thd_log_error("THD engine start failed\n");
 		return THD_ERROR;
 	}
 
-- 
2.7.0.rc3

